test-network-function/operator/suite.go

```diff
@@ -189,18 +191,17 @@ func agrName(operatorName, subName, testName string) string {
 
 //nolint:gocritic // ignore hugeParam error. Pointers to loop iterator vars are bad and `testCmd` is likely to be such.
 func runTestsOnOperator(args []interface{}, name, namespace string, testCmd testcases.BaseTestCase) {
-	ginkgo.When(fmt.Sprintf("under test is: %s/%s ", namespace, name), func() {
-		ginkgo.It(fmt.Sprintf("tests for: %s", testCmd.Name), func() {
-			defer results.RecordResult(identifiers.TestOperatorInstallStatusIdentifier)
-			cmdArgs := strings.Split(fmt.Sprintf(testCmd.Command, args...), " ")
-			opInTest := operator.NewOperator(cmdArgs, name, namespace, testCmd.ExpectedStatus, testCmd.ResultType, testCmd.Action, defaultTimeout)
-			gomega.Expect(opInTest).ToNot(gomega.BeNil())
-			test, err := tnf.NewTest(context.GetExpecter(), opInTest, []reel.Handler{opInTest}, context.GetErrorChannel())
-			gomega.Expect(err).To(gomega.BeNil())
-			gomega.Expect(test).ToNot(gomega.BeNil())
-			testResult, err := test.Run()
-			gomega.Expect(err).To(gomega.BeNil())
-			gomega.Expect(testResult).To(gomega.Equal(tnf.SUCCESS))
-		})
+	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestOperatorInstallStatusIdentifier)
```

Comment:
Same here, I believe we need to use testCmd.Name instead of identifiers.TestOperatorInstallStatusIdentifier

Reply:
fixed